variables.tf

```diff
@@ -84,18 +83,18 @@ variable "msr_volume_size" {
 }
 
 variable "windows_administrator_password" {
-  default     = "w!ndozePassw0rd"
+  default     = ""
```

Comment:
@cranzy We may need to test if an empty password works, if not, we should enforce the user to provide a valid password by defining a validation for the password.

Reply:
ya it does complete fine. It's just that the password field will be missing
`  "hosts":
  - "role": "manager"
    "ssh":
      "address": "3.141.35.201"
      "keyPath": "./ssh_keys/ddd-mke.pem"
      "user": "ubuntu"
  - "role": "worker"
    "winRM":
      "address": "18.225.55.111"
      "insecure": true
      "password": null
      "useHTTPS": true
      "user": "Administrator"
  - "role": "worker"
    "winRM":
      "address": "18.191.172.120"
      "insecure": true
      "password": null
      "useHTTPS": true
      "user": "Administrator"`
      
   But that's on the user to be honest.

Reply:
Resolving and we agreed to have a separate ticket to enforce windows password when windows nodes are created.